controllers/openshift_controller.go

```diff
@@ -345,6 +345,7 @@ func generateDropinConfig(handlerName string) (string, error) {
   runtime_path = "/usr/bin/containerd-shim-kata-v2"
   runtime_type = "vm"
   runtime_root = "/run/vc"
+  privileged_without_host_devices = true
```

Comment:
We might want to make this configurable on the `KataConfig` level and default this to `true` by default. WDYT?

Reply:
@jensfr :-)

Reply:
I guess the use case for privileged_without_host_devices=false will be SR-IOV and possibly other devices down the road. If we need it for those use cases I would make it configurable.

Reply:
We will probably need it for those.  @bpradipt @dgibson WDYT?

Reply:
Hmm, for SRIOV I don't think this change will have an impact. However I would like to test it once. I'll update this thread.

Reply:
lgtm, we can make it configurable in Kataconfig later on. I'll create an issue for it

Reply:
yeah we can make it configurable later on.